kobotoolbox/kpi
# fix(sso): 500 on admin login

### 🗒️ Checklist

1. [x] run linter locally
2. [x] update all related docs (API, README, inline, etc.), if any
3. [x] draft PR with a title `<type>(<scope>)<!>: <title> TASK-1234`
4. [x] tag PR: at least `frontend` or `backend` unless it's global
5. [x] fill in the template below and delete template comments
6. [x] review thyself: read the diff and repro the preview as written
7. [x] open PR & confirm that CI passes
8. [x] request reviewers, if needed
9. [ ] delete this section before merging

### 📣 Summary
Fixes a 500 being thrown when a non-superuser goes to /admin and then tries to log in as a superuser.


### 👷 Description for instance maintainers
Upgrade django-allauth to 65.1.0.


### 💭 Notes
The simplest solution seemed to be just to upgrade django-allauth. It's unclear which exact commit fixes the bug and we are very behind in versions so upgraded to the latest. Nothing obvious broke except one unit test.


### 👀 Preview steps

Bug template:
1. ℹ️ Have 2 accounts, one super and one not
2. Log in as the non-superuser
3. Go to /admin
4. You should be redirected to the login page.
5. Log in as the super user
6. 🔴 [on main] 500 error
7. Checkout the PR branch. Make sure to reinstall requirements.
8. Do 1-5 again.
9. 🟢 [on PR] You should be correctly logged in and sent to the admin page
